WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -317,7 +318,10 @@ public async void ModifiedPasteAsync()
 
 			if (ValidateEntryText(text))
 			{
-				OnTextInput(new TextInputEventArgs { Text = text });
```

Comment:
Big bug from Preview 4, they removed public ctors for these events but was restored in nightlies. cc @nopara74 @wieslawsoltes

Reply:
They will be restored